src/app/infuse/InfusionFinder.scss

```diff
@@ -8,10 +8,6 @@
   max-width: 800px;
   margin: 0 auto;
 
-  @include phone-portrait {
-    --item-margin: 6px;
```

Comment:
This was originally meant to help squeeze an extra item into the grid, not sure if it's needed anymore.